cvc5_pythonic_api/cvc5_pythonic.py

```diff
@@ -1727,6 +1735,545 @@ def Or(*args):
     return _nary_kind_builder(Kind.OR, *args)
 
 
+#########################################
+#
+# Strings and Sequences
+#
+#########################################
+
+class SeqSortRef(SortRef):
+    def cast(self,val):
+        """Try to cast `val` as a String.
+        """
+        if is_expr(val):
+            if debugging():
+                _assert(self.ctx == val.ctx, "Context mismatch")
+            val_s = val.sort()
+            if self.eq(val_s):
+                return val
+            if val_s.is_bool() and self.is_int():
+                return If(val, 1, 0)
+            if val_s.is_bool() and self.is_real():
+                return ToReal(If(val, 1, 0))
+            if val_s.is_int() and self.is_real():
+                return ToReal(val)
+            if debugging():
+                _assert(False, "SMT Integer/Real expression expected")
+        elif type(val) == str:
+            return StringVal(str(val))
+        else:
+            return None
+    
+    def is_string(self):
+        """Determine if sort is a string
+        >>> s = StringSort()
+        >>> s.is_string()
+        True
+        >>> s = SeqSort(IntSort())
+        >>> s.is_string()
+        False
```

Comment:
please make sure that all functions that are called in all these examples also appear in tests.